install/operator/config/Makefile

```diff
@@ -0,0 +1,287 @@
+#
+# Make Options
+#
+MK_OPTIONS := -s
+
+#
+# Allows for resources to be loaded from outside the root location of
+# the kustomize config file. Ensures that resource don't need to be
+# copied around the file system.
+#
+# See https://kubectl.docs.kubernetes.io/faq/kustomize
+#
+KOPTIONS := --load_restrictor LoadRestrictionsNone
+
+#
+# Vars that can be overridden by external env vars
+#
+VERSION ?= 1.12.0
+DEBUG ?= false
+DB_IMAGE ?= centos/postgresql-10-centos7
+DEFAULT_IMAGE ?= quay.io/syndesis/syndesis-operator
+IMAGE ?= $(DEFAULT_IMAGE)
+TAG ?= $(VERSION:.0=)
+NAMESPACE ?= syndesis
+KUBE_USER ?= developer
+LEGACY ?= false
+
+CHANNELS := $(VERSION:.0=.x)
+DEFAULT_CHANNEL := $(VERSION:.0=.x)
+PACKAGE := syndesis
+BUNDLE_DIR := bundle
+PKG := ../pkg
+CRD := crd/bases
+MANAGER := manager
+MANIFESTS := manifests
+GRANT := rbac-grant
+ROLE := rbac-role
+
+#
+# Allows for resources to be loaded from outside the root location of
+# the kustomize config file. Ensures that resource don't need to be
+# copied around the file system.
+#
+# See https://kubectl.docs.kubernetes.io/faq/kustomize
+#
+KOPTIONS := --load_restrictor LoadRestrictionsNone
```

Comment:
This KOPTIONS is already declared above.

Reply:
It outputs error 
```
Error: unknown flag: --load_restrictor
```
From kustomize help, the correct parameter is `--load-restrictor`

Reply:
Version 3 of kustomize uses underscores and this is what was being downloaded by the `kustomize` rule in the Makefile. I've upped the version to 4.0.5 (latest) which uses hyphens instead.